deployment/terraform/firewall.tf

```diff
@@ -0,0 +1,67 @@
+# allow all inbound traffic to the load balancer on port 443
```

Comment:
The security groups for being able to SSH into the bastion do not exist here. There is a security group associated with the bastion (`sg-0cc2163aa727b806f`), but it has no ingress or egress rules.

Reply:
Good catch.

Reply:
There are still no egress rules. `yum` works because I think the packages are hosted on S3 (and we have a VPC endpoint setup for it), but I cannot:

- Connect to the database
- Interact with the ECS services
- Hit any service on the public internet via HTTP/HTTPS